auth/src/main/java/com/firebase/ui/auth/AuthUI.java

```diff
@@ -254,12 +184,8 @@ public Void then(@NonNull Task<GoogleApiClient> task) throws Exception {
      */
     public Task<Void> delete(@NonNull FragmentActivity activity) {
         // Initialize SmartLock helper
-        CredentialTaskApi credentialHelper = GoogleSignInHelper.getInstance(activity);
-
-        return getDeleteTask(credentialHelper);
```

Comment:
Same comment here, I have no issue with inlining this method but it's not relevant to the title of the PR.